o/odoo/build_info.json

```diff
@@ -0,0 +1,11 @@
+{
+    "package_name" : "odoo",
+    "github_url":"https://github.com/odoo/odoo",
+    "version": "16.0",
+    "package_dir": "o/odoo/",
+
+    "16.0" : {
+        "dir":"16.0_debian_bullseye",
+        "build_script" :"odoo_16.0_ubi_8.5.sh"
```

Comment:
@vishakadesai 
As you are using `debian:bullseye-slim` as base image, please provide the `base_docker_image` tag as `debian:bullseye-slim` and `base_docker_variant` as `ubuntu`.
**We need to provide these 2 details for all the non UBI8 based docker images.**